.github/workflows/ci.yml

```diff
@@ -37,7 +37,7 @@ jobs:
     - name: Checkout Code
       uses: actions/checkout@v2
       with:
-          submodules: true
+          submodules: false
```

Comment:
Since there won't be any submodules anymore, there is no need to specify this anymore. Just delete the `with` block entierly.

Reply:
done